package/sqldatabases/definition.yaml

```diff
@@ -93,11 +93,14 @@ spec:
                     pattern: '^([01][0-9]|2[0-3]):([0-5][0-9])-([01][0-9]|2[0-3]):([0-5][0-9])$'
                     default: "00:00-01:00"
                     example: "A valid example: 01:00-02:00"
-                  isNonProduction:
-                    description: Is a non-production database
+                  isProduction:
+                    description: Is a production database
                     type: boolean
+                  finalBackupName:

```

Comment:
shouldn't be required?

Reply:
nope, you don't provide a value for non-production otherwise you get the problem deleting

Reply:
oh dear!

Reply:
just go ahead with the merge otherwise I will jump off of the root top